### DISCUSSION TITLE
Help - How to get multiple alignments for the top hits?

### DISCUSSION THREAD

**Discussion by FelipeMSD** [bbuchfink/diamond#604] [Q&A] [answered]
Dear all,

Is there a way to get on the same row of the output file the names of all entries that equally aligned to a read, receiving the same bitscore and e-value? I'm currently using the option --max-target-seqs 1 in order to filter only the best hit, but this is giving me only one entry (based on alphabetic order), even though there are others with the exact bitscore and e-value.

Thanks in advance for your help.

Kind regards,
Felipe

**Comment by bbuchfink** [comment, marked as answer]
Try using `--top 0`.

**Reply by FelipeMSD** [reply]
Thanks! I'll try!
